<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphTabWidget</class>
 <widget class="QWidget" name="GraphTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="GraphToolbar" name="buttonToolbar" native="true"/>
   </item>
   <item>
    <widget class="QTabWidget" name="editorForms_tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="placeholder_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Выберите или создайте граф для редактирования</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphToolbar</class>
   <extends>QWidget</extends>
   <header>GraphEdit/graphtoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
